crates/proto/src/xfer/dnssec_dns_handle.rs

```diff
@@ -715,7 +715,7 @@ where
         .filter_map(|rrsig|rrsig.into_data())
         .map(|sig| {
             let rrset = Arc::clone(&rrset);
-            let mut handle = handle.clone_with_context();
+            let  handle = handle.clone_with_context();
```

Comment:
Nit: there seems to be a double space here? Not sure why rustfmt isn't catching it.